src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -311,21 +311,26 @@ async def get_feed_tip(query: bytes, autopay: TellorFlexAutopayContract) -> Opti
         msg = "can't suggest feed, autopay contract not connected"
         error_status(note=msg, log=logger.critical)
         return None
-    try:
+
+    if query in CATALOG_QUERY_IDS:
```

Comment:
Why's it named `query`? Isn't the key for this dictionary the stringified `queryId`?

Reply:
there is a comment about this here:  https://github.com/tellor-io/telliot-feeds/blob/10099d7c4d9ffbc2c5831c0f399bfa2cc9b5d761/src/telliot_feeds/reporters/reporter_autopay_utils.py#L306